### DISCUSSION TITLE
Postcss tailwind plugin - filters broken when using @apply

### DISCUSSION THREAD

**Discussion by gtournie** [tailwindlabs/tailwindcss#7528] [Help]
**What version of Tailwind CSS are you using?**

v3.0.23

**What build tool (or framework if it abstracts the build tool) are you using?**

postcss-cli 9.1.0
postcss: 8.4.6

**What version of Node.js are you using?**

v16.3.0

**What browser are you using?**

Chrome

**What operating system are you using?**

macOS

**Reproduction URL**

https://codesandbox.io/s/sveltekit-sandbox-forked-g8p2ll?file=/src/routes/index.svelte

**Describe your issue**

Expected: All filter css-vars (like --tw-blur, --tw-contrast...) should be initialized with a whitespace (e.g. `--tw-blur: ;`)
Issue: When using tailwind postcss plugin, these whitespaces are stripped (`--tw-blur:;`). Thus, filters won't work.

Whitespaces removed by cssnano?

**Comment by adamwathan** [comment]
Hey! Yep this is a bug in whatever minifier you are using, Tailwind is generating a space there and some other tool is removing it. I feel like this should be fine in the latest version of every popular minifier though, so try updating dependencies and seeing if that helps.

Going to convert to a discussion since not a bug in Tailwind.

**Reply by gtournie** [reply]
Well, if you look at the reproduction URL you'll see I don't use any minifier.
If I only use a grayscale class it works though.
Can we reopen this issue? As this definitely comes from tailwindcss package
